rust/gitxetcore/src/data/mdb.rs

```diff
@@ -892,11 +893,20 @@ pub async fn cas_stat_git(config: &XetConfig) -> errors::Result<()> {
     let metas = MDBShardMetaCollection::open(&get_cache_meta_file(&config.merkledb_v2_cache)?)?;
 
     for meta in metas {
+        // For shards before the age of CAS Xorb compression,
+        // `stored_bytes_on_disk` is ZERO. We will report the
+        // non-compressed size for that field.
+        stored_bytes_on_disk += if meta.shard_footer.stored_bytes_on_disk != 0 {
+            meta.shard_footer.stored_bytes_on_disk
+        } else {
+            meta.shard_footer.stored_bytes
+        };
         materialized_bytes += meta.shard_footer.materialized_bytes;
         stored_bytes += meta.shard_footer.stored_bytes;
     }
 
     println!("{{");
+    println!("\"total_compressed_cas_bytes\" : {stored_bytes_on_disk},");
```

Comment:
nit: maybe only print this line if it is different from stored_bytes?

Reply:
Done